models.py

```diff
@@ -44,18 +44,26 @@ def get_missions(self):
 	
 	def get_rank(self, semester=get_current_semester()):
 		from ranks import rank
+		if not semester:
```

Comment:
This 'if' clause looks redundant. semester=get_current_semester should handle this case

Reply:
You would think so.  Apparently if a falsey value is passed, `semester` will not default to `get_current_semester()` because a value was technically passed.  Adding an `if not semester` check seemed like the easiest workaround for that, but I am open to other suggestions.

Reply:
Interesting, I didn't expect that. I don't have any better solutions.